transforms/ember-test-helper-api-migration/README.md

```diff
@@ -0,0 +1,75 @@
+# ember-test-helper-api-migration
+This codemod transfer is to migrate deprecated nameplace `ember-test-helpers` to nameplace `@ember/test-helpers`
```

Comment:
Did you mean "namespace" here? But I think "package" would be more accurate here, wouldn't it?

Reply:
Sure. I will change it to package.